src/EventListener/ExceptionListener.php

```diff
@@ -72,7 +72,7 @@ public function onKernelException(GetResponseForExceptionEvent $event)
         } else {
             switch (substr($code, 0, 1)) {
                 case '4':
-                    $message = 'Input Error';
+                    $message = $exception->getMessage();                    
```

Comment:
I'm not a fan of exposing exception messages by default. An option could be to allow this conditionally, using an injected flag (which could for example be set to the value of kernel.debug).